content/docs/userspace/dist/guide.md

```diff
@@ -0,0 +1,281 @@
+
+If we open a browser now, navigate to `http://localhost:8080` and login with the default fakezod code `lidlut-tabwed-pillex-ridrup`, we'll see our tile's appeared but it says "installing" with a spinner due to the missing glob:
+
+![Installing Tile](https://media.urbit.org/docs/userspace/dist/local-install-1.png)
+
+## Create files for glob
+
+We'll now create the files for the glob. We'll use a very simple static HTML page that just displayes "Hello World!" and an image. Typically we'd have a more complex JS web app that talked to apps on our ship through Eyre's channel system, but for the sake of simplicity we'll forgo that. Let's hop back in the Unix terminal:
+
+```
+[user@host zod]$ cd ~
+[user@host ~]$ mkdir hello-glob
+[user@host ~]$ cd hello-glob
+[user@host hello-glob]$ mkdir img
+[user@host hello-glob]$ wget -P img https://media.urbit.org/docs/userspace/dist/pot.svg
+[user@host hello-glob]$ tree
+.
+└── img
+    └── pot.svg
+
+1 directory, 1 file
+```
+
+We've grabbed an image to use in our "Hello world!" page. The next thing we need to add is an `index.html` file in the root of the folder. The `index.html` file is mandatory; it's what will be loaded when the app's tile is clicked. Let's open our preferred editor and create it:
+
+```
+[user@host hello-glob]$ vim index.html
+```
+
+In the editor, paste in the following HTML and save it:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      div {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <title>Hello World</title>
+  <body>
+    <div>
+      <h1>Hello World!</h1>
+      <img src="img/pot.svg" alt="pot" width="219" height="196" />
+    </div>
+  </body>
+</html>
+```
+
+Our `hello-glob` folder should now look like this:
+
+```
+[user@host hello-glob]$ tree
+.
+├── img
+│   └── pot.svg
+└── index.html
+
+1 directory, 2 files
+```
+
+## Upload to glob
+
+We can now create a glob from the directory. To do so, navigate to `http://localhost:8080/docket/upload` in the browser. This will bring up the `%docket` app's [Globulator](/docs/userspace/dist/glob#globulator) tool:
+
+![Globulator](https://media.urbit.org/docs/userspace/dist/globulator.png)
+
+Simply select the `hello` desk from the drop-down, click `Choose file` and select the `hello-glob` folder in the the file browser, then hit `glob!`.
+
+Now if we return to our ship's homescreen, we should see the tile looks as we specified in the docket file:
+
+![Installed Tile](https://media.urbit.org/docs/userspace/dist/local-install-2.png)
+
+And if we click on the tile, it'll load the `index.html` in our glob:
+
+![Hello World!](https://media.urbit.org/docs/userspace/dist/local-install-3.png)
+
+Our app is working!
+
+## Publish
+
+The final step is publishing our desk with the `%treaty` agent so others can install it. To do this, there's a simple command in the dojo:
+
+```
+> :treaty|publish %hello
+>=
+```
```

Comment:
This is missing the step where you `|public %hello` to make the desk publicly readable. By default, all desks are private.

Reply:
this is unnecessary, treaty implicitly sets it public:

```
  ++  publish
    (poke-our:pass %hood kiln-permission+!>([desk / &]))
```

https://github.com/urbit/urbit/blob/master/pkg/garden/app/treaty.hoon#L319-L320

it would be necessary if this weren't a docket app, perhaps I should add a note about that somewhere

Reply:
Oh right I keep forgetting about that. Certainly noting it's still mandatory for non-docket apps you want to distribute, yes.